### ISSUE TITLE
Replicating sklearn elastic net results in PiML

### ISSUE THREAD

**Issue by xloffree** [SelfExplainML/PiML-Toolbox#26]
Hi,

I trained an elastic net model using sklearn in python (from sklearn.linear_model import ElasticNet). I used the following parameters: ElasticNet(alpha = 0.434, l1_ratio = 0.5) . What can I enter into the L1 and L2 input in the GLM model in PiML to replicate these results (assuming I am using the same data with the same random seed)?

Thanks

**Comment by CnBDM-Su** [comment]
@xloffree Hi, L1 and L2 should both be 0.217.
The example is attached.
![image](https://user-images.githubusercontent.com/26788062/215939371-dfeac03b-209c-4ef3-9e4d-19d455391354.png)

**Comment by xloffree** [comment]
I tried this and when I run model_explain it says that all coefficients are equal to zero. What is going on here? The elastic net worked fine with sklearn and with R. Is there something wrong? 

Thank you

**Comment by CnBDM-Su** [comment]
@xloffree Can you show the example to me? In my test, our GLM result is always same to the sklearn elasticnet.

**Comment by xloffree** [comment]
<img width="1074" alt="Screenshot 2023-07-29 at 7 04 05 PM" src="https://github.com/SelfExplainML/PiML-Toolbox/assets/64742457/2bc0d729-de1f-49f2-983c-68942554ea59">
<img width="934" alt="Screenshot 2023-07-29 at 7 04 17 PM" src="https://github.com/SelfExplainML/PiML-Toolbox/assets/64742457/06346e30-80dc-426c-9502-4453af8420b3">
<img width="787" alt="Screenshot 2023-07-29 at 7 04 28 PM" src="https://github.com/SelfExplainML/PiML-Toolbox/assets/64742457/810dba51-1ec0-4b30-a296-523da0c7d366">
<img width="1068" alt="Screenshot 2023-07-29 at 7 04 51 PM" src="https://github.com/SelfExplainML/PiML-Toolbox/assets/64742457/1858036b-fe6d-4276-b69f-8009966b4d84">
<img width="1002" alt="Screenshot 2023-07-29 at 7 06 26 PM" src="https://github.com/SelfExplainML/PiML-Toolbox/assets/64742457/a7f9f0ab-4e08-4ca3-8755-d4b458792e23">


When using GLMRegressor, I get very different results than from when I use the method shown in the README, which gives a very poor R2 and says all coefficients are 0. I am certain that this is wrong because we have run elastic net on this data in R and python and have gotten similar results that show a much better R2. The results from the glmRegressor look more reasonable. I am trying to use piml to compare many models at once on the same train/test split (the elastic net models with specific parameters included). We would like to use the results from PiML in an upcoming paper, but this problem with PiML's elastic net is preventing us from being able to do so. Please let me know if there is a solution to this.

Thank you
